src/main/java/io/hops/hopsworks/expat/epipe/EpipeRunner.java

```diff
@@ -0,0 +1,138 @@
+/**
+ * This file is part of Expat
+ * Copyright (C) 2021, Logical Clocks AB. All rights reserved
+ *
+ * Expat is free software: you can redistribute it and/or modify it under the terms of
+ * the GNU Affero General Public License as published by the Free Software Foundation,
+ * either version 3 of the License, or (at your option) any later version.
+ *
+ * Expat is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY;
+ * without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
+ * PARTICULAR PURPOSE.  See the GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License along with
+ * this program. If not, see <https://www.gnu.org/licenses/>.
+ *
+ */
+package io.hops.hopsworks.expat.epipe;
+
+import io.hops.hopsworks.expat.elastic.ElasticClient;
+import org.apache.http.HttpHost;
+import org.apache.http.impl.client.CloseableHttpClient;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+
+import java.io.BufferedReader;
+import java.io.File;
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.nio.charset.StandardCharsets;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+
+public class EpipeRunner {
+  private static final Logger LOGGER = LogManager.getLogger(EpipeRunner.class);
+  
+  private static final String REINDEX_PREFIX = "reindex_of = ";
+  private static final String REINDEX_ALL = REINDEX_PREFIX + "all";
+  private static final String PROJECTS_INDEX = "projects";
+  private static final String FEATURESTORE_INDEX = "featurestore";
+  private static final String REINDEX_PROJECTS = REINDEX_PREFIX + PROJECTS_INDEX;
```

Comment:
apparently we use singular form for this one 
https://github.com/logicalclocks/ePipe/blob/master/src/main.cpp#L213
```suggestion
  private static final String REINDEX_PROJECTS = REINDEX_PREFIX + "project";
```

Reply:
fixed